packages/apollo-server-fastify/src/ApolloServer.ts

```diff
@@ -0,0 +1,146 @@
+import { renderPlaygroundPage } from '@apollographql/graphql-playground-html';
+import { Accepts } from 'accepts';
+import {
+  ApolloServerBase,
+  PlaygroundRenderPageOptions,
+} from 'apollo-server-core';
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { IncomingMessage, OutgoingMessage, Server } from 'http';
+import { processRequest as processFileUploads } from '@apollographql/apollo-upload-server';
```

Comment:
Slight tweaking w/r/t to #2054.

Reply:
https://github.com/apollographql/apollo-server/pull/1971/commits/19d96320ddf295ff4b9d24d8359eb17490b9855d